src/commands/deploy/buildImageInAzure/BuildFromProjectListStep.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep, AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../../utils/localize";
+import { IDeployBaseContext } from "../IDeployBaseContext";
+import { AcrListStep } from "../deployFromRegistry/acr/AcrListStep";
+import { BuildImageStep } from "./BuildImageStep";
+import { DockerFileItemStep } from "./DockerFileItemStep";
+import { IBuildImageInAzureContext } from "./IBuildImageInAzureContext";
+import { ImageNameStep } from "./ImageNameStep";
+import { OSPickStep } from "./OSPickStep";
+import { RootFolderStep } from "./RootFolderStep";
+import { RunStep } from "./RunStep";
+import { TarFileStep } from "./TarFileStep";
+import { UploadSourceCodeStep } from "./UploadSourceCodeStep";
+
+const buildImageInAzure = 'buildImageInAzure';
+export class BuildFromProjectListStep extends AzureWizardPromptStep<IDeployBaseContext> {
+    public async prompt(context: IDeployBaseContext): Promise<void> {
+        const buildFromProjectLabels: string[] = [
+            localize('azure', 'Build from project remotely using Azure Container Registry')
+            //localize('docker', 'Build from project locally using Docker')
+        ];
+
+        if (buildFromProjectLabels.length === 1) {
```

Comment:
This should go in `configureBeforePrompt`. That way your `shouldPrompt` will make this prompt get skipped and we'll never go into `prompt`. This will mess up the total number of steps which isn't that weird, but that is what `configureBeforePrompt` is made for.